email_template_qweb/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+========================
+QWeb for email templates
+========================
+
+This module was written to allow you to write email templates in QWeb instead
```

Comment:
In order to avoid the word write 2 times, you can put directly "This module allows you to write...", but it's not mandatory to change it.